selendroid-standalone/src/main/java/io/selendroid/standalone/server/model/AbstractDeviceStore.java

```diff
@@ -0,0 +1,197 @@
+package io.selendroid.standalone.server.model;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Predicate;
+import com.google.common.base.Strings;
+import com.google.common.collect.FluentIterable;
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+import io.selendroid.common.SelendroidCapabilities;
+import io.selendroid.common.device.DeviceTargetPlatform;
+import io.selendroid.standalone.android.AndroidApp;
+import io.selendroid.standalone.android.AndroidDevice;
+import io.selendroid.standalone.android.AndroidEmulator;
+import io.selendroid.standalone.android.AndroidEmulatorPowerStateListener;
+import io.selendroid.standalone.android.impl.AbstractAndroidDeviceEmulator;
+import io.selendroid.standalone.exceptions.AndroidDeviceException;
+import io.selendroid.standalone.exceptions.DeviceStoreException;
+
+import java.util.*;
+import java.util.logging.Logger;
+
+public abstract class AbstractDeviceStore implements DeviceStore {
```

Comment:
Did you change any code here or is it just a move/copy? Asking because I do have some comments on parts of the code, but if you didn't change it I'd rather keep it as is for now.